sdk/web-pubsub/web-pubsub-client-protobuf/package.json

```diff
@@ -111,7 +111,7 @@
     "mock-socket": "^9.1.5",
     "util": "^0.12.1",
     "@types/ws": "^7.4.5",
-    "protobufjs-cli": "1.0.2",
+    "protobufjs-cli": "1.1.1",
```

Comment:
@zackliu is it necessary to pin the version of the CLI tool, or is it possible to use a `^` version so that we can upgrade its version in lock file regularly?

Reply:
Yep, I think we can use the `^` for cli tool. But for `protobufjs` package, we should use `~`